src/serlo-editor-repo/store/helpers.ts

```diff
@@ -1,139 +1,15 @@
 import * as R from 'ramda'
 import { createSelectorCreator, defaultMemoize } from 'reselect'
 
-import { Action, InternalAction } from './actions'
-import {
-  ScopedState,
-  Selector,
-  ActionCreatorWithPayload,
-  ActionCreatorWithoutPayload,
-  InternalScopedState,
-  InternalSelector,
-  InternalState,
-} from './storetypes'
+export const createDeepEqualSelector = createSelectorCreator(defaultMemoize, {
+  resultEqualityCheck: R.equals,
+  maxSize: 50,
```

Comment:
I don't see any downside either, and I'd love if you'd get involved in the Redux code straight away, as I think it would give you an opportunity to get even more ideas for improvement 😄 Please feel free to add those changes to this PR, or as an improvement in a next PR.